internal/api/websocket.go

```diff
@@ -0,0 +1,79 @@
+package api
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/gorilla/websocket"
+	core "github.com/iden3/go-iden3-core"
+	"github.com/polygonid/sh-id-platform/internal/core/domain"
+	"github.com/polygonid/sh-id-platform/internal/core/ports"
+	"github.com/polygonid/sh-id-platform/internal/log"
+	"net/http"
+	"time"
+)
+
+const Ticker = 12
```

Comment:
better define the duration here or name the value "TickerSeconds" or smth like that